Test build failing at HEAD

```
e: java.lang.ClassCastException: class com.sun.tools.javac.tree.JCTree$JCTypeApply cannot be cast to class com.sun.tools.javac.tree.JCTree$JCArrayTypeTree (com.sun.tools.javac.tree.JCTree$JCTypeApply and com.sun.tools.javac.tree.JCTree$JCArrayTypeTree are in module jdk.compiler of loader 'app')
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.visitVarDef(Pretty.java:691)
	at org.jetbrains.kotlin.kapt3.PrettyWithWorkarounds.visitVarDef(Kapt3Extension.kt:388)
	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCVariableDecl.accept(JCTree.java:972)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:194)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:209)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExprs(Pretty.java:226)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExprs(Pretty.java:234)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.visitMethodDef(Pretty.java:609)
	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCMethodDecl.accept(JCTree.java:866)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:194)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStat(Pretty.java:215)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStats(Pretty.java:242)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printBlock(Pretty.java:318)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.visitClassDef(Pretty.java:578)
	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCClassDecl.accept(JCTree.java:774)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:194)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStat(Pretty.java:215)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStats(Pretty.java:242)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printBlock(Pretty.java:318)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.visitClassDef(Pretty.java:578)
	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCClassDecl.accept(JCTree.java:774)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:194)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStat(Pretty.java:215)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printUnit(Pretty.java:391)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.visitTopLevel(Pretty.java:421)
	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCCompilationUnit.accept(JCTree.java:529)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printExpr(Pretty.java:194)
	at jdk.compiler/com.sun.tools.javac.tree.Pretty.printStat(Pretty.java:215)
	at org.jetbrains.kotlin.kapt3.Kapt3ExtensionKt.prettyPrint(Kapt3Extension.kt:363)
	at org.jetbrains.kotlin.kapt3.AbstractKapt3Extension.saveStubs(Kapt3Extension.kt:325)
	at org.jetbrains.kotlin.kapt3.AbstractKapt3Extension.generateKotlinSourceStubs(Kapt3Extension.kt:310)
	at org.jetbrains.kotlin.kapt3.AbstractKapt3Extension.analysisCompleted(Kapt3Extension.kt:177)
	at org.jetbrains.kotlin.kapt3.ClasspathBasedKapt3Extension.analysisCompleted(Kapt3Extension.kt:106)
	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM$analyzeFilesWithJavaIntegration$2.invoke(TopDownAnalyzerFacadeForJVM.kt:115)
	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM.analyzeFilesWithJavaIntegration(TopDownAnalyzerFacadeForJVM.kt:125)
	at org.jetbrains.kotlin.cli.jvm.compiler.TopDownAnalyzerFacadeForJVM.analyzeFilesWithJavaIntegration$default(TopDownAnalyzerFacadeForJVM.kt:99)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler$analyze$1.invoke(KotlinToJVMBytecodeCompiler.kt:257)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler$analyze$1.invoke(KotlinToJVMBytecodeCompiler.kt:42)
	at org.jetbrains.kotlin.cli.common.messages.AnalyzerWithCompilerReport.analyzeAndReport(AnalyzerWithCompilerReport.kt:115)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.analyze(KotlinToJVMBytecodeCompiler.kt:248)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.compileModules$cli(KotlinToJVMBytecodeCompiler.kt:88)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinToJVMBytecodeCompiler.compileModules$cli$default(KotlinToJVMBytecodeCompiler.kt:47)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:167)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:53)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:101)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:47)
	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:101)
	at org.jetbrains.kotlin.incremental.IncrementalJvmCompilerRunner.runCompiler(IncrementalJvmCompilerRunner.kt:485)
	at org.jetbrains.kotlin.incremental.IncrementalJvmCompilerRunner.runCompiler(IncrementalJvmCompilerRunner.kt:131)
	at org.jetbrains.kotlin.incremental.IncrementalCompilerRunner.doCompile(IncrementalCompilerRunner.kt:424)
	at org.jetbrains.kotlin.incremental.IncrementalCompilerRunner.compileImpl(IncrementalCompilerRunner.kt:360)
	at org.jetbrains.kotlin.incremental.IncrementalCompilerRunner.compileNonIncrementally(IncrementalCompilerRunner.kt:242)
	at org.jetbrains.kotlin.incremental.IncrementalCompilerRunner.compile(IncrementalCompilerRunner.kt:98)
	at org.jetbrains.kotlin.daemon.CompileServiceImplBase.execIncrementalCompiler(CompileServiceImpl.kt:625)
	at org.jetbrains.kotlin.daemon.CompileServiceImplBase.access$execIncrementalCompiler(CompileServiceImpl.kt:101)
	at org.jetbrains.kotlin.daemon.CompileServiceImpl.compile(CompileServiceImpl.kt:1746)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
> Task :ground:kaptGenerateStubsDebugUnitTestKotlin FAILED
FAILURE: Build failed with an exception.
```

Could be related to #1774?

--- a/ground/src/test/java/com/google/android/ground/persistence/local/room/converter/GeometryTest.kt
+++ b/ground/src/test/java/com/google/android/ground/persistence/local/room/converter/GeometryTest.kt
@@ -87,6 +87,6 @@ class GeometryTest {
 
   private fun multiPolygon(vararg polygons: Polygon) = MultiPolygon(polygons.asList())
 
-  private fun toCoordinateList(path: Path): List<Coordinate> =
+  private fun toCoordinateList(path: Path): List<Coordinates> =
     path.map { Coordinates(it.first, it.second) }
 }

--- a/ground/src/test/java/com/google/android/ground/persistence/remote/firebase/schema/GeometryConverterTest.kt
+++ b/ground/src/test/java/com/google/android/ground/persistence/remote/firebase/schema/GeometryConverterTest.kt
@@ -209,7 +209,7 @@ class GeometryConverterTest {
 
   private fun multiPolygon(vararg polygons: Polygon) = MultiPolygon(polygons.asList())
 
-  private fun toCoordinateList(path: Path): List<Coordinate> =
+  private fun toCoordinateList(path: Path): List<Coordinates> =
     path.map { Coordinates(it.first, it.second) }
 
   private fun indexedGeoPointMap(path: Path): Map<String, Any> =

--- a/ground/src/test/java/com/google/android/ground/system/GeocodingManagerTest.kt
+++ b/ground/src/test/java/com/google/android/ground/system/GeocodingManagerTest.kt
@@ -44,7 +44,7 @@ import org.robolectric.ParameterizedRobolectricTestRunner
 class GeocodingManagerTest(
   private val expectedAreaName: String,
   private val message: String,
-  private val inputs: List<Pair<Coordinate, List<Address>>>,
+  private val inputs: List<Pair<Coordinates, List<Address>>>,
 ) : BaseHiltTest() {
   @BindValue @Mock lateinit var geocoder: Geocoder
   @Inject lateinit var geocodingManager: GeocodingManager
@@ -155,10 +155,10 @@ class GeocodingManagerTest(
     private fun testCase(
       expectedAreaName: String,
       message: String,
-      vararg areas: Pair<Coordinate, List<Address>>
+      vararg areas: Pair<Coordinates, List<Address>>
     ) = arrayOf(expectedAreaName, message, areas.toList())
 
-    private fun addresses(coordinates: Coordinate, vararg addresses: Address) =
+    private fun addresses(coordinates: Coordinates, vararg addresses: Address) =
       Pair(coordinates, addresses.toList())
   }
 }

--- a/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/point/LatLngConverterTest.kt
+++ b/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/point/LatLngConverterTest.kt
@@ -17,7 +17,7 @@
 package com.google.android.ground.ui.datacollection.tasks.point
 
 import com.google.android.ground.model.geometry.Coordinates
-import com.google.android.ground.ui.datacollection.tasks.point.LatLngConverter.processCoordinate
+import com.google.android.ground.ui.datacollection.tasks.point.LatLngConverter.processCoordinates
 import com.google.common.truth.Truth.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
@@ -28,7 +28,8 @@ class LatLngConverterTest {
 
   @Test
   fun testProcessCoordinates_ne() {
-    assertThat(processCoordinates(Coordinates(10.555, 10.555))).isEqualTo("10°33'18\" N 10°33'18\" E")
+    assertThat(processCoordinates(Coordinates(10.555, 10.555)))
+      .isEqualTo("10°33'18\" N 10°33'18\" E")
   }
 
   @Test

--- a/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/polygon/PolygonDrawingTaskFragmentTest.kt
+++ b/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/polygon/PolygonDrawingTaskFragmentTest.kt
@@ -134,13 +134,13 @@ class PolygonDrawingTaskFragmentTest :
   }
 
   /** Overwrites the last vertex and also adds a new one. */
-  private fun updateLastVertexAndAddPoint(coordinate: Coordinate) {
+  private fun updateLastVertexAndAddPoint(coordinate: Coordinates) {
     updateLastVertex(coordinate, false)
     onView(withText("Add point")).perform(click())
   }
 
   /** Updates the last vertex of the polygon with the given vertex. */
-  private fun updateLastVertex(coordinate: Coordinate, isNearFirstVertex: Boolean = false) {
+  private fun updateLastVertex(coordinate: Coordinates, isNearFirstVertex: Boolean = false) {
     val threshold = PolygonDrawingViewModel.DISTANCE_THRESHOLD_DP.toDouble()
     val distanceInPixels = if (isNearFirstVertex) threshold else threshold + 1
     viewModel.updateLastVertexAndMaybeCompletePolygon(coordinate) { _, _ -> distanceInPixels }

--- a/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/polygon/PolygonDrawingViewModelTest.kt
+++ b/ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/polygon/PolygonDrawingViewModelTest.kt
@@ -174,13 +174,13 @@ class PolygonDrawingViewModelTest : BaseHiltTest() {
   }
 
   /** Overwrites the last vertex and also adds a new one. */
-  private fun updateLastVertexAndAdd(coordinate: Coordinate) {
+  private fun updateLastVertexAndAdd(coordinate: Coordinates) {
     updateLastVertex(coordinate, false)
     viewModel.addLastVertex()
   }
 
   /** Updates the last vertex of the polygon with the given vertex. */
-  private fun updateLastVertex(coordinate: Coordinate, isNearFirstVertex: Boolean = false) {
+  private fun updateLastVertex(coordinate: Coordinates, isNearFirstVertex: Boolean = false) {
     val threshold = DISTANCE_THRESHOLD_DP.toDouble()
     val distanceInPixels = if (isNearFirstVertex) threshold else threshold + 1
     viewModel.updateLastVertexAndMaybeCompletePolygon(coordinate) { _, _ -> distanceInPixels }

